src/flows/components/header/index.tsx

```diff
@@ -30,7 +31,7 @@ const FlowHeader: FC = () => {
         <RenamablePageTitle
           onRename={handleRename}
           name={flow.name}
-          placeholder="Name this Flow"
+          placeholder={`Name this ${PROJECT_NAME}`}
```

Comment:
nit: `Flow` is capitalized, but `PROJECT_NAME` isn't. Is that gonna be a problem?

Reply:
that is the product request, to specifically make it "notebooks with a lower case n"

Reply:
🚲 🏠 .  looks odd for it to be the only lowercased thing in the sidenav.  idk not a designer.